src/Command/InstallRecipesCommand.php

```diff
@@ -36,15 +36,20 @@ public function __construct(/* cannot be type-hinted */ $flex, string $rootDir)
 
     protected function configure()
     {
-        $this->setName('symfony:sync-recipes')
-            ->setAliases(['sync-recipes', 'fix-recipes'])
+        $this->setName('symfony:recipes:install')
+            ->setAliases(['symfony:sync-recipes', 'sync-recipes', 'fix-recipes'])
             ->setDescription('Installs or reinstalls recipes for already installed packages.')
+            // TODO : Separate the force option in an update command
```

Comment:
I'm not sure about this: if we add an update command, it might be totally different than the current behavior of install --force